### DISCUSSION TITLE
Samsung TV Plus links for Viasat Explore, History, and Nature?

### DISCUSSION THREAD

**Discussion by fraudiay79** [iptv-org/community#1018] [Q&A]
Has anyone found the Samsung TV Plus links for Viasat Explore, History, and Nature?  They have been added to Samsung TV Plus Channels in Scandanavia

https://www.samsung.com/se/smart-tv/samsung-tv-plus/
https://www.samsung.com/no/smart-tv/samsung-tv-plus/
https://www.samsung.com/dk/smart-tv/samsung-tv-plus/
https://www.samsung.com/fi/smart-tv/samsung-tv-plus/

I already checked Matt Huisman's website, he does not have them yet.

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
